src/codemagic/apple/resources/enums.py

```diff
@@ -200,6 +211,39 @@ def from_profile_type(cls, profile_type: ProfileType) -> CertificateType:
         else:
             raise ValueError(f"Certificate type for profile type {profile_type} is unknown")
 
+    @classmethod
+    def resolve(
```

Comment:
It's not quite obvious what we resolve. Consider naming it like `list_expected_certificate_types`. Or just add a docstring with a brief explanation...

Reply:
Updated in [eca517b](https://github.com/codemagic-ci-cd/cli-tools/pull/415/commits/eca517be666856c4f1f7b3f50cceb08bdc61ee00).